iOS/Multiplatform Redux Sample/Multiplatform Redux Sample/Main/MainCoordinator.swift

```diff
@@ -47,6 +47,7 @@ class MainCoordinator: SubCoordinator, Coordinator {
 
     private func handleSettingsNavigation(_ lastScreen: MainScreen, _ navController: UINavigationController) {
         if lastScreen == MainScreen.settings {
+            (navController.viewControllers.last as? Poppable)?.poppingViaAction = true
```

Comment:
Is it possible to invert the behaviour: the default value is true and just when recognizing the swipe-gesture setting it to false?
-> I see the problem, when we add some new Screens navigating as the Settings-VC, that it needs to be handled there as well.

Reply:
We can only find out whether we actually navigate back after the gesture has already ended. The best we could do would be to reset the flag after the gesture has ended with a delay of e.g. 10ms. I don't think there is much of a problem with the current solution, though, since any screens we add in the future would also be `PresenterViewController`s.